scripts/zones/Beaucedine_Glacier/npcs/Parledaire_RK.lua

```diff
@@ -14,11 +16,48 @@ local guardRegion = xi.region.FAUREGANDI
 local guardEvent  = 32763
 
 entity.onTrade = function(player, npc, trade)
+    local zoneId = player:getZoneID()
+    local text = zones[zoneId].text
     xi.conq.overseerOnTrade(player, npc, trade, guardNation, guardType)
+    if
+        player:getNation() == guardNation or
+        xi.settings.GARRISON_NATION_BYPASS == 1
+    then
+        xi.garrison.onTrade(player, npc, trade)
+    else
+        --not of nation event
+        player:messageSpecial(text.GARRISON, trade:getItem():getID(), player:getNation())
+    end
 end
 
 entity.onTrigger = function(player, npc)
-    xi.conq.overseerOnTrigger(player, npc, guardNation, guardType, guardEvent, guardRegion)
+    local zoneId = npc:getZoneID()
+    local text = zones[zoneId].text
+    local win = player:getZone():getLocalVar(string.format("[GARRISON]Treasure_%s", zoneId))
+    local won = player:getCharVar("Garrison_Won")
```

Comment:
Could these be condensed down into a single `GarrisonStatus` var?

Reply:
needs better naming:
win is when the trader has won the garrison but has yet to trigger the win with the guard.
won is after the trader has triggered the win other players can now talk to npc to remove effects and get the won text.

needs 2 different vars though for the 2 different triggers

Reply:
fixed in #1919